### PR TITLE
Solution & Project Update

### PR DESCRIPTION
**Pull request by ennerperez**
In order to be more consistent with new dotnet project, I suggest some of this solution/project enhancements.

### CODE REVIEW

File: CompilePalX/Updating/UpdateManager.cs [hunk 1]
```diff
@@ -54,14 +43,20 @@ static UpdateManager()
             RegistryManager.Write("Version", currentVersionString);
             RegistryManager.Write("PrereleaseVersion", currentPrereleaseVersionString);
         }
+        public static string CurrentVersion => currentVersion.ToString(isPrerelease ? 2 : 1);
+        public static string LatestVersion => latestVersion.ToString(isPrerelease ? 2 : 1);
+        // Tags must be in form: v0major.minor
+        private static string PrereleaseUpdateURL => $"https://github.com/ruarai/CompilePal/releases/tag/v0{LatestVersion}";
+        public static Uri UpdateURL => new Uri(isPrerelease ? PrereleaseUpdateURL : MajorUpdateURL);
+        public static event UpdateFound OnUpdateFound;
```
Comment (Exactol) [inline CompilePalX/Updating/UpdateManager.cs:51]: Move fields before methods

**Review by Exactol — CHANGES_REQUESTED** [review]
Thank you for cleaning up the code. Everything looks good, but I had a few comments.

Personally I'm not a fan of using var instead of explicit types for most cases, except if the type appears on the right hand side of the assignment. It's fine when using an IDE, but it makes it harder to read if you use a text editor that doesn't display the types, like when looking at the code on GitHub.

File: CompilePalX/Compilers/BSPPack/AssetUtils.cs [hunk 2]
```diff
@@ -343,10 +382,20 @@ public static List<string> findMdlRefs(string path)
...
+            {
+                ".dx80.vtx",
+                ".dx90.vtx",
+                ".phy",
+                ".sw.vtx",
+                ".vtx",
+                ".xbox.vtx",
+                ".vvd",
+                ".ani"
+            };
```
[the remark this answers is not in the document]
Comment (ennerperez) [reply CompilePalX/Compilers/BSPPack/AssetUtils.cs:395]: image

Let's turn explicit types then, sounds good.

**Comment by ennerperez** [comment]
I have to close this and create new PRs with minor changes to avoid significant conflicts in the future.

**Pull request closed without merging**